assets/css/custom.css

```diff
@@ -69,21 +80,82 @@ body, #qcSiteNav h4, #qcPriNav ul > li > a, #qcPriNav ul ul > li > a, .dash_titl
 
 /* ASSIGN CUSTOM PAGE BACKGROUND COLOR/IMAGE */
 
-body.home.static {
+.page-intro {
 	background: url('../img/WSC-bg.jpg') no-repeat center center fixed;
    -webkit-background-size: cover;
    -moz-background-size: cover;
    -o-background-size: cover;
    background-size: cover;
+	 padding-bottom: 3em;
+	 border-bottom: 5px solid #EB6851;
 }
-body.home:after {
-	background: none;
+
+.page-intro #qcOpenNav {
+    background: url(../img/theme/openNavInvert.png) center center;
 }
 
 #qcSlideText .qcTargetLink {
 	color: #EB6851;
 }
 
-#qcWidgets, .qcOrderedList.check li:before {
+/* SPONSORS SECTION STYLES */
+.sponsors-section {
+	padding: 3em 0;
+}
+
+.sponsors-section h2 {
+	text-transform: uppercase;
+	letter-spacing: 2px;
+	color: #172751;
+	font-size: 40px;
+}
+
+.sponsors-section h3 {
+	font-size: 28px;
+	color: #EB6851;
+	border-bottom: 2px solid #EB6851;
+	margin-top: 16px;
+	margin-bottom: 16px;
+	padding-bottom: 8px;
+}
+
+.sponsors-section p {
+	line-height: 1.5em;
+	margin-bottom: 16px;
+}
+
+.sponsors-section img {
+	width: 100%;
+}
+
+.sponsors-section .prospectus-link {
 	background-color: #172751;
+	border-radius: 8px;
+	color: white;
+	padding: 8px 16px;
+	display: inline-block;
+	text-transform: uppercase;
+	letter-spacing: 2px;
+	line-height: 1.5;
+}
+
+@media (max-width:1022px) {
+	.sponsors-section .prospectus-link {
+		margin-top: 16px;
+		margin-bottom: 8px;
+		font-size: 12px;
+		padding: 16px;
+	}
+}
+
+.sponsors-section .prospectus-link:hover {
+	background-color: #EB6851;
+}
```

Comment:
Ok cool will do, I was going to ask about making some global classes, but I wasn't sure how people would want to handle design elements, so I figured I'd make everything specific to the sponsor section at first.